README.md

```diff
@@ -1,7 +1,30 @@
+# Figma Plugin API typings
+[![npm](https://img.shields.io/npm/v/@figma/plugin-typings?logo=npm&cacheSeconds=1800)](https://www.npmjs.com/package/@figma/plugin-typings)
+
 This repository contains the typings for the Figma Plugin API.
 
-It is intended to be installed as "@figma/plugin-typings" and the types should become globally available without needing to use import statements. We do it this way because the plugin API is part of the host environment, as opposed to being a package that a plugin includes.
```

Comment:
This is kind of important. For the vast majority of typings, which are typings for libraries, you need to import the type definitions that you want to use (since you're importing code from the library after all). This is one of the few typings where it makes sense to do something different.

Reply:
Not very 😉 
nobody imports typing files into the project, except some edge cases. Two ways how typings can be delivered:
1. `npm i -D @types/packagename` - TS autoresolve this path, without additional import
2. `npm i packagename` - TS files already inside this package

and in both of these cases without any imports, so it's not something specific and should be noted

Reply:
I don't...think that's true? I forget if there's some way to configure TypeScript not to require explicit import statements, but some configurations do. For example, `{ "module": "commonjs", "moduleResolution": "node" }`.

In large codebases, there's typically a preference towards needing explicit import statements for everything (including type definitions) because if you don't do that, definitions that have the same name could clash (loudly or worse, silently), whether between two project files or between a project file and a definition in one of the dependencies.

Reply:
You described edge case, and it's not relevant for Figma plugins, because it's casual and simple projects without some custom/monorepo/conflict names stuff 🤷